q2cli/core/usage.py

```diff
@@ -47,6 +47,10 @@ def write_plugin_example_data(plugin, output_dir):
 class CLIUsageVariable(usage.UsageVariable):
     EXT = {
         'artifact': '.qza',
+        # it would be nice to have a / as the 'ext' for collections so that
+        # it's clear that it's a directory, but that's proving to be a pain
+        # so putting on hold for now
```

Comment:
Agreed, what seems to be the issue with that?

Reply:
![Screen Shot 2023-07-06 at 11 21 42 AM](https://github.com/qiime2/q2cli/assets/54517601/892e383e-c331-4e3c-950d-a2a58f013e6f)

@ebolyen this is what happens when adding '/' as the extension for result_collection, and including '/' after the input and output collection dirs within the usage tests. We weren't confident that there was a straightforward way to handle, but this would definitely be the ideal solution.

Reply:
It's probably pretty easy, the checker lives on the Click Type, which would have enough context to know if it was a collection, and change this check as appropriate: https://github.com/qiime2/q2cli/blob/master/q2cli/click/type.py#L96-L98

Reply:
I think it's actually the line above what I linked that over-interprets the trailing slash

Reply:
FWIW, this is actually something a user would run into as well, because if they added a reasonable trailing slash, they would get the same error, so this is actually the usage API giving us a better range of tests.